extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpBuildTimeConfig.java

```diff
@@ -31,29 +31,41 @@
     public boolean virtual;
 
     /**
-     * The HTTP root path for non application endpoints. Various endpoints such as metrics, health,
-     * and open api are deployed under this path.
-     * Setting the value to "/" disables the separate non application root,
-     * resulting in all non application endpoints being served from "/" along with the application.
+     * A common root path for non-application endpoints. Various extension-provided endpoints such as metrics, health,
+     * and openapi are deployed under this path by default.
+     *
+     * * Relative path (Default, `q`) ->
+     * Non-application endpoints will be served from
+     * `${quarkus.http.root-path}/${quarkus.http.root-path}`.
```

Comment:
`${quarkus.http.root-path}/${quarkus.http.non-application-root-path}` ?

Reply:
Yes, but the path is defined by both the values, not `quarkus.http.root-path` repeated twice, I assume?
My point is that you're repeating `quarkus.http.root-path` twice here

Reply:
aha, now I see what you mean.

Will fix the description

Reply:
you're right. cut-paste error that none of our eyeballs caught. ;)